src/modules/test/recipes/mof-to-json.awk

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/awk -f
```

Comment:
Also here:

# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.